.travis.yml

```diff
@@ -7,14 +7,6 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
-    - env: TOXENV=py27
-      python: 2.7
-    - env: TOXENV=py27-pinned
-      python: 2.7
-    - env: TOXENV=py27-extra-deps
-      python: 2.7
-    - env: TOXENV=pypy
```

Comment:
in `install` section there is code related to pypy

Reply:
@kmike indeed, what should we do about that section? Just remove the `"$TOXENV" = "pypy"` part?

Reply:
Remove the whole `if … fi` section.

Reply:
You could also remove the `pypy3` part, and use `python: pypy3` in the matrix (see https://github.com/scrapy/w3lib/pull/140)

Reply:
#4116